pkgs/games/gscrabble/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, gtk3, gobjectIntrospection, wrapGAppsHook, gst_all_1
+, python3, python3Packages, gnome3, hicolor-icon-theme }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gscrabble";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "RaaH";
+    repo = "gscrabble";
+    rev = "4609e6be4867836afa98dd1d2c4f5296d9706416";
+    sha256 = "0j6102ylzwlqc2bvswff49m8psbi3qikv63m95ps8sr8yhwi2idp";
+  };
+
+  doCheck = false;  # no tests available
+
+  buildInputs = with gst_all_1; [
+    gst-plugins-base gst-plugins-good gst-plugins-ugly gst-plugins-bad
+  ];
+
+  nativeBuildInputs = [ 
+    python3
+    gobjectIntrospection # populate GI_TYPELIB_PATH
+    wrapGAppsHook
+   ];
+
+  propagatedBuildInputs = with python3Packages; [
+    gtk3
+    gst-python
+    pygobject3
+  ];
+
+  propagatedUserEnvPkgs = [
```

Comment:
You should rely on `wrapGAppsHook` picking it up from `buildInputs`. `propagatedUserEnvPkgs` is an ugly hack that will hopefully get removed soon.